base/reducedim.jl

```diff
@@ -45,7 +45,7 @@ end
 initarray!(a::AbstractArray{T}, f, ::Union{typeof(min),typeof(max),typeof(_extrema_rf)},
     init::Bool, src::AbstractArray) where {T} = (init && mapfirst!(f, a, src); a)
 
-for (Op, initval) in ((:(typeof(&)), true), (:(typeof(|)), false))
+for (Op, initval) in ((:(typeof(and_all)), true), (:(typeof(or_any)), false))
```

Comment:
Does this break `missing` support?

Reply:
Never mind, `missing` already doesn't work with `dims`

Reply:
Seems like it'd be good to support at some point.  Open an issue?